### PR TITLE
Update zh_CN.lang

### PR DESCRIPTION
**Pull request by NealDeal34**

### CODE REVIEW

File: src/main/resources/assets/angelica/lang/zh_CN.lang [hunk 1]
```diff
@@ -115,14 +118,16 @@ options.iris.reset.tooltip.holdShift=按住 SHIFT 点击以重置
...
 options.iris.mustDisableFullscreen=请先禁用全屏！
-options.iris.setToDefault=把选项设为默认值？
+options.iris.setToDefault=将选项设为默认值？
 options.iris.profile=配置文件
 options.iris.profile.custom=自定义
 options.iris.shadowDistance=最大阴影距离
-options.iris.shadowDistance.enabled=允许更改阴影的最远距离。超出此距离的地形和实体不会投射阴影。降低阴影距离可以显著提高性能。
-options.iris.shadowDistance.disabled=你不能改变当前的光影包已经为阴影设置的渲染距离。
+options.iris.shadowDistance.enabled=允许更改渲染阴影的最远距离。超出此距离的地形和实体不会投射出阴影。减少阴影渲染距离可以显著提高性能。
+options.iris.shadowDistance.disabled=你不能改变当前光影包已经为阴影设置的渲染距离。
```
Comment (Nxer) [inline src/main/resources/assets/angelica/lang/zh_CN.lang:126]: `当前光影包设置了阴影距离，无法修改。` 改成人话

**Review by Nxer — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/angelica/lang/zh_CN.lang [hunk 2]
```diff
@@ -115,14 +118,16 @@ options.iris.reset.tooltip.holdShift=按住 SHIFT 点击以重置
...
 options.iris.gui.hide=隐藏 GUI
 options.iris.gui.show=显示 GUI
+options.dynamic_lights=动态光源
+options.dynamic_lights.tooltip=启用动态光源. 模式选择: OFF - 关闭, FASTEST - 500ms延迟, FAST - 250ms延迟, FANCY - 无延迟. 如果正在使用光影则此功能被禁用.
```
Comment (Hikari1nVoid) [inline src/main/resources/assets/angelica/lang/zh_CN.lang:130]: `如果正在使用光影则此功能被禁用`-->`使用光影时此选项无效`
weird grammar...

**Review by Hikari1nVoid — CHANGES_REQUESTED** [review]
这么个长句没个逗号是否合适

File: src/main/resources/assets/angelica/lang/zh_CN.lang (hunk 2 above)
Comment (NealDeal34) [reply src/main/resources/assets/angelica/lang/zh_CN.lang:130]: Honestly, do you really know the differences between 'feature is disabled' and 'the state can't be changed'?

**Comment by mitchej123** [comment]
Will merge when a (native) Chinese Speaker approves - doesn't need to be a dev

**Comment by Dream-Master** [comment]
@Kiwi233

File: src/main/resources/assets/angelica/lang/zh_CN.lang [hunk 3]
```diff
@@ -47,7 +48,7 @@ sodium.options.use_block_face_culling.tooltip=如果启用，则只会渲染面
 sodium.options.use_compact_vertex_format.name=使用压缩顶点格式
 sodium.options.use_compact_vertex_format.tooltip=如果启用，将使用更紧凑的顶点格式来渲染方块。这可以显著降低显存使用量和带宽要求，尤其是集显，但由于降低了位置和纹理坐标属性的精度，可能会与一些资源包发生冲突。
 sodium.options.use_fog_occlusion.name=使用雾遮挡剔除
-sodium.options.use_fog_occlusion.tooltip=如果启用，确定被雾效果完全遮挡的块将不会被渲染，有助于提高性能。当雾效果较重时（例如在水下时），改进可能会更加显著，但在某些情况下可能会导致天空和雾之间出现不良的视觉瑕疵。
+sodium.options.use_fog_occlusion.tooltip=如果启用，确定被雾效果完全遮挡的块将不会被渲染，有助于提高性能。当雾效果较重时（例如在水下时），提高可能会更加显著，但在某些情况下可能会导致天空和雾之间出现不良的视觉瑕疵。
```
Comment (Hikari1nVoid) [inline src/main/resources/assets/angelica/lang/zh_CN.lang:51]: please revert this one.original one sounds way more better.

File: src/main/resources/assets/angelica/lang/zh_CN.lang [hunk 4]
```diff
@@ -61,8 +62,8 @@ sodium.options.cpu_render_ahead_limit.tooltip=指定CPU可以在GPU上等待完
 sodium.options.cpu_render_ahead_limit.value=%s 帧
 sodium.options.allow_direct_memory_access.name=允许直接内存访问
 sodium.options.allow_direct_memory_access.tooltip=如果启用，将允许某些关键代码路径使用直接内存访问来提高性能。这通常会大大减少区块和实体渲染的 CPU 开销，但会使诊断某些错误和崩溃变得更加困难。只有您被要求这么做或知道在做什么时，才应该禁用它。
-sodium.options.ignore_driver_blacklist.name=忽略驱动程序黑名单
-sodium.options.ignore_driver_blacklist.tooltip=如果启用，将忽略与硬件/驱动程序配置已知的不兼容，允许启用可能导致游戏问题的选项。你平时不应该碰这个选项，除非你确切地知道自己在做什么。更改此选项后，必须保存、关闭并重新打开设置屏幕。
+sodium.options.ignore_driver_blacklist.name=忽略驱动程序冲突
```
Comment (Hikari1nVoid) [inline src/main/resources/assets/angelica/lang/zh_CN.lang:65]: 驱动程序黑名单 is a widely accepted when describing drivers with vulnerabilities and need to be blocked/blacklisted.
original is better

File: src/main/resources/assets/angelica/lang/zh_CN.lang [hunk 5]
```diff
@@ -111,18 +114,20 @@ options.iris.shaders.nonePresent=光影：无光影包
 options.iris.back=返回
 options.iris.reset=重置
 options.iris.reset.tooltip=重置所有选项并应用？
-options.iris.reset.tooltip.holdShift=按住 SHIFT 点击以重置
+options.iris.reset.tooltip.holdShift=按住 SHIFT 并点击以重置
```
Comment (Hikari1nVoid) [inline src/main/resources/assets/angelica/lang/zh_CN.lang:117]: original one is better.remember this is a tooltip.

**Review by Hikari1nVoid — COMMENTED** [review]
Please make sure the translation is compliant with local language culture.Otherwise spamming DeepL translator is sufficient.

File: src/main/resources/assets/angelica/lang/zh_CN.lang (hunk 4 above)
Comment (NealDeal34) [reply src/main/resources/assets/angelica/lang/zh_CN.lang:65]: Did you just search for what '驱动程序黑名单' means? Or, as you say,it is used to describe 'vulnerabilities' or with unsafe signs that should be disabled, not the term 'incompatible' in this context.And it's cause misunderstand for players.Don't just literal translation“blacklist”，ok?

Comment (NealDeal34) [reply src/main/resources/assets/angelica/lang/zh_CN.lang:65]: Did you just search for what '驱动程序黑名单' means? Or, as you say,it is used to describe 'vulnerabilities' or with unsafe signs that should be disabled, not the term 'incompatible' in this context.And it's cause misunderstand for players.Don't just literal translate “blacklist”，ok?

**Comment by NealDeal34** [comment]
> Please make sure the translation is compliant with local language culture.Otherwise spamming DeepL translator is sufficient.

I didn't mean to offend you, but please don't make me question your Chinese proficiency, ok？It's obvious which one was machine translation,if you can't get it,perhaps we can seek clarification from someone else :)

**Comment by Dream-Master** [comment]
@Nxer

**Comment by Hikari1nVoid** [comment]
all good on commit e4128e01e54ff6f05212158e36c66d24807a9a66

**Review by mitchej123 — APPROVED** [review]

**Review by Nxer — APPROVED** [review]
No wrong words.

**Pull request merged**